### PR TITLE
[WM-2019] Method details in Submission Config page (in Workflows tab)

### PR DESCRIPTION
**Pull request by salonishah11**
### What
- Migration of the summary details portion of Submission Config page from CBAS UI

### Why
- Migrating CBAS UI into Terra UI

### Testing strategy
- Existing CBAS-UI test files copied over corresponding to the respective components/pages (only parts of SubmissionConfig.test.js were copied as the rest will be added in follow up tickets)

In browser:
- Enter Azure workspace + Launch Cromwell app
- Enable Workflows feature flag in console: configOverridesStore.set({ isAzureWorkflowsTabVisible: true })
- Refresh and click on Workflows tab
- Page should be populated with cards for 3 Covid-19 workflows
- Click on any card and it should load the Submission Config page with method details. Each tab in the page will be migrated in follow up tickets

https://github.com/DataBiosphere/terra-ui/assets/16748522/bcbded7d-4dd6-4935-b485-9ed015b81510

### CODE REVIEW

**Review by dillydally414 — APPROVED** [review]
Do we have a ticket to incorporate the remaining tests for this page? (Not the ones that will be covered in 2020-2022, the ones under "submitting a run set" [here](https://github.com/DataBiosphere/cbas-ui/blob/5f19dab4a3f1c559904d9672704aa76e7f6a0d61/src/pages/SubmissionConfig.test.js#L1964)) If not, we should create that as a reminder so we do not lose those tests in the migration.

**Comment by salonishah11** [comment]
@dillydally414 that's a good point. I have created https://broadworkbench.atlassian.net/browse/WM-2083 for migrating the tests related to "submitting a run set".

**Comment by salonishah11** [comment]
Regarding code coverage - I have tried to add as many tests I could for the changes. There are few flows which aren't tested because they are still left to be migrated in tickets WM-2020 to WM-2022 and that should bring the Code Coverage to 80%+.

**Comment by salonishah11** [comment]
Updated Helpful Links Box to fix the color contrast accessibility issue

Screenshot 2023-07-13 at 1 01 13 PM

File: src/workflows-app/SubmissionConfig.js [hunk 1]
```diff
@@ -1,9 +1,504 @@
...
+              ['select-data', () => h2('TODO')], // https://broadworkbench.atlassian.net/browse/WM-2020
+              ['inputs', () => h2('TODO')], // https://broadworkbench.atlassian.net/browse/WM-2021
+              ['outputs', () => h2('TODO')] // https://broadworkbench.atlassian.net/browse/WM-2022
+            ),
+          ]
+        ),
+      ]);
+};
+
+export const SubmissionConfig = wrapWorkflowsPage({ name: 'SubmissionConfig' })(BaseSubmissionConfig);
```
Comment (mspector) [inline src/workflows-app/SubmissionConfig.js:501]: 👍

**Review by mspector — APPROVED** [review]

**Pull request merged**